src/components/Sidebar/cart/CartSection.test.js

```diff
@@ -38,4 +39,16 @@ describe('CartSection', () => {
 
     expect(wrapper.find(Section).prop('leftHeaderContent')).toBe('schedule, schedule');
   });
+
+  it('renders modal correctly', () => {
```

Comment:
Noticed this just now - I think a more descriptive test name would be something along the lines of "opens modal on click"